directus-aws/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'How to Deploy Directus to AWS EC2'
+description: 'An how to guide on how to host a self hosted instance of Directus on AWS, use s3 for file storage and AWS '
+author:
+  name: 'Trust Jamin'
+  avatar_file_name: 'jamin.png'
+---
+
+## Introduction
+
+In this tutorial, you will learn how to deploy a self-hosted instance of Directus to Amazon Web Services (AWS) EC2, connect it to an AWS RDS PostgreSQL database and S3 storage bucket.
+
+## Before You Start
+
+You will need:
+
+- An Amazon Web Service account ([AWS](https://aws.amazon.com)) with access to its `SecretKey` and `AccessKey`.
+- A Directus project - using the [self-hosted guide](https://docs.directus.io/getting-started/quickstart.html).
+
+## Set up an AWS RDS Database
+
+Login to your AWS account and head to the [RDS page](https://eu-west-2.console.aws.amazon.com/rds/home) or search for RDS on the search bar to create a new database.
+
+In the Create new database page,  select a standard create for database creation method, and on the engine options, select `PostgreSQL` (Directus also supports other databases such as, `MySQL`, `OracleDB`, `And Microsoft SQL` on AWS RDS).
+
+![An AWS RDS database creation page with options to configure the database](create-db.png)
+
+In the settings options, create a name for your database instance and a username and password credentials to log in to the database (You'll use these credentials to log in via Directus).
+
+![An AWS RDS database creation page with options to configure the settings and login method of the database](db-settings.png)
+
+- On the connectivity options, choose __Don't connect to an EC2 compute resource__ (this is because you haven't created an EC2 instance yet), and choose the default virtual private cloud (VPC) for connecting to the database.
+
+- Select the `No` option for public access to the database to ensure the database can only be connected via the VPC security firewall.
+
+- For the VPC security group (firewall), choose the default security group and select password authentication for database authentication.
+
+![An AWS RDS database creation page to set up the connectivity options to the database](db-connection.png)
```

Comment:
Note to self: Once again, for no additional info and burden of updating screenshots, remove.